selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/testbase/LocatorTestBase.java

```diff
@@ -105,11 +109,11 @@
           By.xpath("//*[@class='button primary' and @value='Log In']");
 
   // SSO Login with CAS
-  public static final By ELEMENT_INPUT_PASSWORD_CAS = By.id("password");
+  public static final By ELEMENT_INPUT_PASSWORD_CAS = By.id("UIPortalLoginFormControl");
 
   public static final By ELEMENT_INPUT_USERNAME_CAS = By.id("username");
 
-  public static final By ELEMENT_SIGN_IN_BUTTON_CAS = By.xpath(".//*[contains(@class,'btn-submit')]");
+  public static final SelenideElement ELEMENT_SIGN_IN_BUTTON_CAS = $(byClassName("button"));
```

Comment:
"button" class name is probably used in others locations in the DOM, it seems dangerous to rely on it. Can't we do the selector in a subcontainer of the wiki instead of the whole page ?